### DISCUSSION TITLE
Alien Ruins Overhaul

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#8492] [Ideas and suggestions]
_Note: I've worked a lot on this since this was posted and summed it all up on the mod page_

**https://steamcommunity.com/sharedfiles/filedetails/?id=2825448171**
 ![image](https://user-images.githubusercontent.com/73229309/184945217-276dd4f8-81e8-49ab-b464-85a9e89779ad.png)

 
List of issues

### Level design
**Ruins can be cut through**, both walls and doors can be broken, bypassing any "puzzle" or making navigation trivial.
**It's easy to get stuck** on some passages, you often have to swim backwards and ragdoll through.

### Enemies:
**Guardians damage values are inconsistent** Damage multiplier for head is 1.25, while every other limb is 0.65 to 0.35 meaning damage dealt is very inconsistent. Guardian2 steam cannon will melt anyone who's not in a PUCS in 3 seconds.
**Guardians can't navigate** tight spaces and can be cheesed by getting them stuck.

### Puzzles
**Terminal "puzzles" are extremely simplistic**. Put curio in terminal, press both buttons for good measure, solve the *entirety* of ruins puzzles/contraptions
**Rooms are static**. You only need to see a room once and know exactly what to do.

### Rewards
**Ruins break the economy**. You often get 1 Ruin per cave and the loot tables are hysterical, add Xenologist medics and ruins are worth thousands and thousands of marks. Safer than mining materials in abyss islands too.
**Alien Artifacts do nothing**. Artifact transport case nullifies the artifact effect, lets you carry several artifacts at once. Or you can negate Psychosis/Sky/Thermal artifacts by tossing them in the ballast 

------

This is an attempt to make ruins more enjoyable, using a new philosophy.

### New ruins design guidelines:

1. Guardians are the main threat. 
2. The ruins layout does not work against the guardian.
3. Every trap is a danger multiplier in presence of the guardian
4. Terminals can be used to disable hazards temporarily
5. Terminals used to access rewards activate hazards
6. No puzzle where uncessary
7. Doors and walls cannot be destroyed
8. Ruins must be rare and dangerous

_Minor tweaks (may or may not be good):
-Less doors (ruins are less clicky)
-Less terminals (less inventory management with curios)
-Less motion detectors_ _(hopefully better performance)_

----

Also see: #8479

**Comment by LePiromano** [comment]
i agree on pretty much everything except buffing guardians, your average jimmy is just gonna shoot blindly with an smg and accomplish nothing

**Comment by GenericGoose** [comment]
current ruins can be defined by 2 words: annoying and unrewarding
you get a small amount of alien toys, alien weapons suck, big artifacts dont spawn outside of missions (https://github.com/Regalis11/Barotrauma/issues/7233)
i skip them more often than not
puzzles are boring and should just be removed entirely
there's that shock trap thing that seems to do nothing at all
narrow passages are obnoxious, constantly getting stuck in them
harpoon guardian jitter your ragdoll, probably a multiplayer issue
ruins need more enemies to spawn in them, more enemy variants, more traps and bring the artifacts back
destructible walls are a necessary feature to cut down on the annoyance
I also think alien guns need a rework, I never bothered using them before, but now that you have all the perk weapons and engineer making Free Infinite Ammo, forever, out of fuel rods that also work as ammo, why would you ever bother?

**Reply by cshingle** [reply]
> current ruins can be defined by 2 words: annoying and unrewarding

I totally agree with this. I generally skip them for this reason. Whenever I "have" to do them I'll just load up some rail gun rounds with oxygenite shards and punch a deep hole in it to avoid some of the tediousness.

The alien pistol is just a fancy plasma cutter. It is just another trinket to sell.

**Reply by LePiromano** [reply]
akimbo alien pistols have  82 dps without talents, i don't know what you mean by that unless you mean the ancient weapon
